doc/workshops/spyder_workshop_2_astro.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Scientific Computing and Visualization with Spyder
+
+Astrological signs and cognitive ability
+
+Created on Thu May 20 10:17:27 2021
+
+@author: Spyder Team
+"""
+
+# %% Import the libraries
+import scipy.stats as stats
+import pandas as pd
+import numpy as np
+
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+
+# %% Load raw data (csv) and save it in parquet format:
+data = pd.read_csv("parsed_data_public.csv", low_memory=False)
+data.to_parquet("parsed_data_public.parquet")
+
+
+# %% Load raw data (parquet)
+data = pd.read_parquet("parsed_data_public.parquet")
```

Comment:
This has the same problem as above, i.e. we should use only one way to load the workshop data instead of two.